modules/seqexec/server/src/main/scala/seqexec/server/ODBSequencesLoader.scala

```diff
@@ -53,35 +54,41 @@ final class ODBSequencesLoader[F[_]: ApplicativeError[?[_], Throwable]: Logger](
 
         // Verify that the program id is valid
         configObsId
-          .adaptErr{ case _ => SeqexecFailure.Unexpected(s"Invalid $configObsId")}
-          .flatMap( s => ApplicativeError[F, Throwable].catchNonFatal(SPProgramID.toProgramID(s)))
-          .flatMap( _ => translator.sequence(seqId, odbSeq))
+          .adaptErr { case _ => SeqexecFailure.Unexpected(s"Invalid $configObsId") }
+          .flatMap(s => ApplicativeError[F, Throwable].catchNonFatal(SPProgramID.toProgramID(s)))
+          .flatMap(_ => translator.sequence(seqId, odbSeq))
       }
 
     def loadSequenceEvent(seqg: SequenceGen[F]): EventType[F] =
       Event.modifyState[F, EngineState[F], SeqEvent]({ st: EngineState[F] =>
         st.sequences
           .get(seqId)
-          .fold(ODBSequencesLoader.loadSequenceEndo(seqId, seqg, execEngine))(
-            _ => ODBSequencesLoader.reloadSequenceEndo(seqId, seqg, execEngine)
+          .fold(ODBSequencesLoader.loadSequenceEndo(seqId, seqg, execEngine))(_ =>
+            ODBSequencesLoader.reloadSequenceEndo(seqId, seqg, execEngine)
           )(st)
       }.withEvent(LoadSequence(seqId)).toHandle)
 
     t.map {
-      case (err :: _, None) =>
-        List(Event.logDebugMsgF[F, EngineState[F], SeqEvent](explain(err)))
-      case (errs, Some(seq)) =>
+      case (UnrecognizedInstrument(_) :: _, None) =>
```

Comment:
I don't know whether we need this, but is an unrecognized instrument logged at least once somewhere else or not at all anymore?

Reply:
That would be ideal but it forces to keep an state and I want to avoid that.
In reality this log was useful during development but not anymore.
This function is called every 30s thus polluting the log